doc/Programming-guide/Key-platform-features/Data-collection/Rest-log-appender/index.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: page
+title: Rest log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Rest-log-appender
+sort_idx: 90
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+
+
+* TOC
+{:toc}
+
+The REST log appender is responsible for transferring logs from Operation server to your custom service.
+
+# Creating REST log appender with Admin UI
+
+The easiest way to create a Cassandra log appender for your application is by using Admin UI.
+
+To create a log appender for access via REST, do the following:
+
+1. In the **Log appenders** window, click **Add log appender**.
+2. Enter the log appender name and description, select the minimum and maximum supported log schema version, and select necessary log metadata fields.
+3. Set the log appender type to _REST_.
+4. Fill in other fields as required.
+5. Click **Add** button. Log appender is ready and operational at this point.
+
+![Add log appender in Admin UI](attach/add-log-appender-in-admin-ui.png)
+
+
+It's also possible to create REST appender using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
```

Comment:
Why that information is not a part of this page as it is done for other log appenders?